website/src/templates/markdown.js

```diff
@@ -216,18 +216,25 @@ const Link = props => {
     );
   }
 
-  // At this point, `href` values are relative but the link router expects
-  // a full relative path from the base URL path.
-  // E.g. if `href="./foo"` and we're on page `/getting-started/bar`, the
-  // value we need is `/getting-started/foo`.
-  // To achieve that, we let the DOM API build the full URL, then we simply
-  // extract the relative path.
-  const linkElement = document.createElement('a');
-  linkElement.href = props.href;
-  const absoluteUrl = linkElement.href; // <-- this now is the full absolute URL
-  const [, relativePath] = absoluteUrl.split(window.location.origin);
+  // Since `document` is not available in SSR, we need to replace
+  // the link with the proper value once the component is mounted.
+  const [linkTo, setLink] = React.useState(props.href);
+  React.useEffect(() => {
+    // At this point, `href` values are relative but the link router expects
+    // a full relative path from the base URL path.
+    // E.g. if `href="./foo"` and we're on page `/getting-started/bar`, the
+    // value we need is `/getting-started/foo`.
+    // To achieve that, we let the DOM API build the full URL, then we simply
+    // extract the relative path.
+    const linkElement = document.createElement('a');
+    linkElement.href = props.href;
+    const absoluteUrl = linkElement.href; // <-- this now is the full absolute URL
+    const [, relativePath] = absoluteUrl.split(window.location.origin);
+    setLink(relativePath);
+  });
```

Comment:
Don't have have to specify `[props.href]` as a dep? Should we add the ESLint rules for hooks on the pkg?

Reply:
The prop won't change, so it shouldn't be necessary. Or am I missing something?

Reply:
In that case it's fine. It's just underspecifying the dependencies which I read is rarely what one wants and can lead to quite silent bugs.

Reply:
Yeah I know. I can add an empty array if you want, to make it "safer"?

Reply:
I think that's the same. Anyway. It's fine just wanted to make sure.